### PR TITLE
Cleanup workflows

### PR DESCRIPTION
**Pull request by jsoref**
Fix cleanup.yml regressions from
https://github.com/prestodb/presto/pull/17285

Test plan
None

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: .github/workflows/kudu.yml [hunk 1]
```diff
@@ -14,8 +14,6 @@ env:
 jobs:
   kudu:
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
```
Comment (arunthirupathi) [inline .github/workflows/kudu.yml:18]: Any reason for removing this ?

**Review by arunthirupathi — COMMENTED** [review]
Thanks for making this change.

File: .github/workflows/product-tests-specific-environment.yml [hunk 2]
```diff
@@ -14,8 +14,7 @@ env:
 jobs:
   product-tests-specific-environment1:
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
+    name: environment 1
```
Comment (arunthirupathi) [inline .github/workflows/product-tests-specific-environment.yml:17]: May be remove this change, on the tests on this PR there are two hung tests.

Comment (jsoref) [reply .github/workflows/product-tests-specific-environment.yml:17]: It's the same thing as this:
https://github.com/prestodb/presto/pull/17384/files#r817057495

If something is hanging, then setting a shorter timeout (keeping in mind how long the task should run) would be a better approach.

**Review by arunthirupathi — APPROVED** [review]
Thanks for this contribution. There is a flaky test failure, so will merge this change.

**Pull request merged**